whatshap/cli/polyphase.py

```diff
@@ -992,12 +992,13 @@ def compute_linkage_based_block_starts(readset, pos_index, ploidy, single_linkag
             continue
         q = Queue()
         q.put(i)
+        merged_clust[i] = new_num_clust
```

Comment:
In general, it would be best to have separate commits for unrelated changes, but keep it as it is now.